static/js/apps/eval_retrieval_generation/sxs/left_right_picker.ts

```diff
@@ -0,0 +1,25 @@
+import sdbm from "sdbm";
+
+import { DocInfo } from "../types";
+
+/**
+ * Result is stable based on inputs.
```

Comment:
generally the function comment should describe what the function is doing. So maybe also add something like "Gets the docInfo to show in the left and right panes."

Reply:
Revised comment.